### ISSUE TITLE
Navigation thought dropdown items are not readable for the screen reader

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud-libraries/nextcloud-vue#2756]
Dropdown provides a possibility to select the option via keyboard. Unfortunately, they don’t work well with screen readers. When pressing the down arrow to move down to the select options the focus moves visually, but the screen reader only reads any text written in the field or “Empty” if the field is indeed empty.

ToDo: fix `VueMultiselect.vue`

**Comment by susnux** [comment]
Fixed as we only have vue-select (`NcSelect`) nowadays and removed NcMultiselect
